Dockerfile

```diff
@@ -13,8 +13,10 @@ RUN go mod download
 COPY main.go main.go
 COPY pkg/ pkg/
 
+ARG TARGETARCH
```

Comment:
dont we need something like this ?
https://github.com/kubevirt/bridge-marker/pull/77/files#diff-5aecd73bbcb9b7410938673269f0d4e676dd18509277b0ff2f2787c9d3e22604R3-R11

Reply:
Not really, we only need `TARGETARCH` and `ARG` already sets it as an enviroment variable.

Reply:
Thanks

@ashokpariya0 does it mean maybe that the logic on bridge-marker or whatever projects that done the same as it can be simplified maybe?

Reply:
@oshoval No, it's important to handle default cases. The combination of `ARG` and `ENV` is beneficial because it allows you to manage both build-time and runtime configurations more flexibly and efficiently.